packages/core/src/Stores/client-store.js

```diff
@@ -1296,6 +1296,9 @@ export default class ClientStore extends BaseStore {
         const is_samoa_account = this.root_store.ui.real_account_signup_target === 'samoa';
         let currency = '';
         form_values.residence = this.residence;
+        if (!form_values.tax_residence) {
```

Comment:
why is this required?

Reply:
BE needs this value to do a check on tin_skipped